network_importer/version.py

```diff
@@ -1,3 +1,3 @@
 """Version number for network_importer."""
-__version_info__ = (3, 0, "2")
+__version_info__ = (3, 0, "3")
```

Comment:
I'd love to see us just get this from the Poetry settings at some point, as I'm pretty sure I would not have done this in the past if it was me.

Reply:
agreed